reference/channels.v3.yml

```diff
@@ -452,11 +452,9 @@ paths:
                 1.product_id: error.path.missing
```

Comment:
Remove the changes to this file from the pr.

Reply:
When I remove it, I receive a message saying the yml file is invalid.